.golangci.yml

```diff
@@ -104,18 +104,20 @@ linters-settings:
       - BUG
       - FIXME
       - HACK
+  gosec:
+    exclude-generated: true
   lll:
     line-length: 120
+  misspell:
+    locale: US
+    ignore-words: expect
   nolintlint:
     allow-leading-space: false
     require-explanation: true
     require-specific: true
-  unparam:
```

Comment:
Did you manually re-order these or was this automatic in an editor/linter/etc?

Reply:
this was manual, but there are tools to automatically sort lines and structures

Reply:
https://plugins.jetbrains.com/plugin/14407-smartsort